nvflare/app_common/statistics/numeric_stats.py

```diff
@@ -15,17 +15,18 @@
 from math import sqrt
 from typing import Dict, List, TypeVar
 
-from tdigest import TDigest
-
 from nvflare.app_common.abstract.statistics_spec import Bin, BinRange, DataType, Feature, Histogram, HistogramType
 from nvflare.app_common.app_constant import StatisticsConstants as StC
-from nvflare.app_common.statistics.statistics_config_utils import get_target_percents
+from nvflare.app_opt.statistics.quantile_stats import get_quantiles
```

Comment:
This file "numeric_stats.py" is in app_common, should not include file in "app_opt".
We could move this "numeric_stats.py" to app_opt